static/js/timerender.js

```diff
@@ -171,7 +171,10 @@ export function render_date(time, time_above, today) {
 export function render_markdown_timestamp(time, text) {
     const hourformat = page_params.twenty_four_hour_time ? "HH:mm" : "h:mm a";
     const timestring = format(time, "E, MMM d yyyy, " + hourformat);
-    const titlestring = "This time is in your timezone. Original text was '" + text + "'.";
+    const titlestring =
+        "This time is in your timezone. Original text was '" +
+        text +
+        "'. Click to generate the calendar file!";
```

Comment:
It doesn't feel right to append `Click to generate the calendar file` text in the existing tooltip. Would love to get other suggestions :)

Reply:
Can make this text as bootstrap-tooltip ? Currently, it is just a title which displays on hover and wait 3s vs just hover to display.

For the text, we can move "Original text was ..." to the end.

"This time is in your timezone. Add to your calendar by opening the file downloaded by clicking on this. Original ..."